apps/web/src/state/usePasscodeAuth/usePasscodeAuth.ts

```diff
@@ -1,50 +1,32 @@
 import { useCallback, useEffect, useState } from 'react';
 import { useHistory, useLocation } from 'react-router-dom';
 import { Location } from 'history';
+import { apiClient } from '../api/api';
 
-const endpoint = process.env.REACT_APP_TOKEN_ENDPOINT || '/token';
+const endpoint = '/verify-passcode';
```

Comment:
Don't think we need a variable for this. I think this value is just used in one spot. 

`return fetch('verify-passcode', {`